apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -169,7 +168,8 @@ const styles = {
     marginBottom: 5
   },
   buttonWithMargin: {
-    marginRight: '5px'
+    margin: 0,
```

Comment:
what's the reason for this change?

Reply:
Glad you asked! Since it'll be `button` instead of a `div` it'll have `margin: 5px; `style from `application.scss` button base style. This line is needed to overwrite base style and keep the button appearance be the same it was before changing `div` to `button`